README.md

```diff
@@ -16,15 +37,19 @@ certificates, you can visit https://hubs.local:4000 from your browser.
 
 1. [Install Docker Compose](https://docs.docker.com/compose/install)
 2. [Install Mutagen Compose](https://github.com/mutagen-io/mutagen-compose#system-requirements)
-3. Add these entries to your hosts file:
+3. [Install Mutagen](https://mutagen.io/documentation/introduction/installation)
+4. Add these entries to your hosts file:
 
         127.0.0.1   hubs.local
         127.0.0.1   hubs-proxy.local
 
-4. Initialize the services with `bin/init`
+  - On Windows, your plain-text `hosts` file is probably located at `C:\Windows\System32\drivers\etc\hosts`.
+5. Initialize the services with `bin/init`
 
 ### Orchestration
 
+First, ensure the Mutagen daemon is running with `mutagen daemon start`.
```

Comment:
What would you think about adding `mutagen daemon start` to `bin/up` in lieu of this note?

Reply:
That seems reasonable to me. Do you think we should also stop the Mutagen daemon in `bin/down`?

The only reason I can think of **not** to add `mutagen daemon stop` to `bin/down` is to ensure that Hubs-related code doesn't impact other applications that the person is running on their computer.

WDYT?

Reply:
I would leave it out of `bin/down`. Your reason _not_ to add `stop` is reason enough. It’s necessary to have the daemon started but not necessary to have it stopped.

Reply:
Comment addressed; please review!